projects/imx6/options

```diff
@@ -50,20 +50,23 @@
 
   # Kernel to use.
   # default is 4.4 from xbian
-    if [ "$LINUX_VERSION" = "sr-3.14" ]; then
-      LINUX="imx6-3.14-sr"
+    if [ "$LINUX_VERSION" = "3.14-sr" ]; then
+      LINUX="imx6-$LINUX_VERSION"
 
       # OpenGL-ES implementation to use
       OPENGLES="imx-gpu-viv"
-    elif [ "$LINUX_VERSION" = "xbian-4.8" ]; then
-      LINUX="imx6-4.8-xbian"
-      LIBCEC_TYPE="xbian"
+    elif [ "$LINUX_VERSION" = "4.8-xbian" ]; then
+      LINUX="imx6-$LINUX_VERSION"
+
+      # OpenGL-ES implementation to use
+      OPENGLES="gpu-viv-bin-mx6q"
+    elif [ "$LINUX_VERSION" = "4.10-xbian" ]; then
```

Comment:
Fyi: it is 4.10.8 and seems to be working with le master. CEC doesn't work and needs investigation.
It could also be used for 8.0 just like 4.4.